### DISCUSSION TITLE
What is a jurisdiction?

### DISCUSSION THREAD

**Discussion by mother10** [FamilySearch/GEDCOM#483] [Q&A] [answered]
This might seem a really strange question, because there are definitions of Jurisdiction online. They say something like:
"a country, state, or other area where a particular set of laws or rules must be obeyed".
In the GEDCOM I see no definition anywhere.

For a PLAC it is obvious that City, State etcetera are jurisdictions.
But now what for the smallest one, the leftmost one.
I thought that was Hamlet, or Gehucht or Hameaux. So something with just a couple of houses.

But I heard people say that it could also be a house or a castle or a church.
So they give a PLAC as for instance:
PLAC CityLane 123,Myvillage, Myprovince, Somecountry
So the  first one is in fact an address. Its a street with a housenumber.

Would that be correct?
Should there be a definition in the GEDCOM for jurisdiction?

**Comment by tychonievich** [comment, marked as answer]
Discussed in steering committee meeting 18 JUN 2024

Jurisdiction is not defined in the specification, and all of the cases you shared can be seen as following one or more of the definitions of the word "jurisdiction." We have also seen people use cemeteries, buildings, and farms as jurisdictional entries. All are in accordance with the specification as written.

Several large applications have chosen to use a fixed set of jurisdictional types with the smallest unit being a city/town/village/hamlet, so many of the examples may follow that pattern, but there is nothing in the spec limiting PLAC to that usage or the number of levels of a PLAC. Always use a PLAC.FORM (or HEAD.PLAC.FORM) to communicate the jurisdictions in use.

In discussing this, we also noticed two gaps in the current documents:

- The general FAQ should address this topic.
- The spec has a gap in its wording; if a PLAC has not FORM and there's not HEAD.PLAC.FORM then the definition of PLAC gives it no meaning. We should fix that ambiguity.

**Comment by Norwegian-Sardines** [comment]
Jurisdiction very often refers to “law making”, but in reality it is more about the “authority” or “control” an entity has over the aspects of a particular concept/thing/domain.  As a programmer with specific knowledge of a piece of software, I was the only one “authorized” to make changes to code relating to that piece of software domain.  Similarly the Postal Service is the only entity with authority to create/change addresses (their domain), a Church or Cemetery is the only entity authorize to manage the affairs within its domain.

one definition that talks about jurisdictional authority is:
“the limits or territory within which authority may be exercised”

- So a `BIRT`.`PLAC` tag can include the hospital name for people that were born in that hospital, due to their authority to have births at their location or domain.
- A `BURI`.`PLAC` can contain the cemetery name based on their authority in that domain.
- A `RESI`.`PLAC` can contain an address based on the authority of the PO in that domain.

This is how I see and use the term jurisdiction in this case!

**Comment by mother10** [comment]
Thanks for both of the answers!
